src/mgmt/srv_pool.c

```diff
@@ -482,9 +533,6 @@ ds_mgmt_create_pool(uuid_t pool_uuid, const char *group, char *tgt_dev,
 	}
 out_uuids:
 	D_FREE(tgt_uuids);
-tgt_fail:
-	if (rc)
-		ds_mgmt_tgt_pool_destroy(pool_uuid);
```

Comment:
So what if pool destroy failed after tgt_pool_create_ranks()? I assume it still needs to destroy the pool?

Reply:
Fixed